src/client/java/net/neoforged/neoforge/client/model/quad/BakedColors.java

```diff
@@ -0,0 +1,58 @@
+package net.neoforged.neoforge.client.model.quad;
+
+public sealed interface BakedColors {
```

Comment:
Can we have some basic class javadoc, explaining that the colors are either per-quad or per-vertex? And mentioning that they are in 0xaabbggrr (iirc?) format.